components/Common/CodeBox/__tests__/index.test.mjs

```diff
@@ -0,0 +1,102 @@
+import * as Toast from '@radix-ui/react-toast';
+import { render, screen, waitFor, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { IntlProvider } from 'react-intl';
+
+import CodeBox from '@/components/Common/CodeBox';
+
+jest.mock('@heroicons/react/24/outline', () => ({
+  CodeBracketIcon: jest.fn(() => <svg data-testid="code-bracket-icon" />),
+  DocumentDuplicateIcon: jest.fn(() => (
+    <svg data-testid="document-duplicate-icon" />
+  )),
+}));
+
+const originalNavigator = { ...window.navigator };
+
+const messages = {
+  'components.common.codebox.copy': 'copy',
+};
+
+describe('CodeBox', () => {
+  const Sut = props => {
+    const {
+      language = 'javascript',
+      content = 'const value = 10;',
+      showLineNumbers,
+    } = props;
+
+    return (
+      <Toast.Provider>
+        <IntlProvider locale="en" messages={messages}>
```

Comment:
We don't recommend passing IntlProvider on tests. We recommend passing our own `LocaleProvider`